src/controllers/initgamecontroller.js

```diff
@@ -133,6 +133,11 @@ export default class InitGameController extends Controller
             )
         });
 
+        // join room 
+        $("#open-create-team-modal").click((e) => {

```

Comment:
This should be registered in the `ScoreboardController`.

Reply:
Somehow this is not working.

Reply:
Where is the modal ID you reference added? Is it part of the view rendered by the `ScoreboardController`? It should be registered in the controller that sets-up this item (I think this might be the sidebar, so it should be registered with that thing).

Reply:
No nvm this is already fine. Github gives me a strange view on the file. Don't mind this.